test/fixtures/rules/std_rule_cases/L066.yml

```diff
@@ -0,0 +1,75 @@
+rule: L066
+
+test_pass_no_config:
+  pass_str: |
+    select
+      x.a,
+      x_2.b
+    from x
+    left join x as x_2 on x.foreign_key = x.foreign_key
+
+test_fail_alias_too_short:
+  fail_str: |
+    SELECT
+      u.id,
+      c.first_name,
+      c.last_name,
+      COUNT(o.user_id)
+    FROM users as u
+    JOIN customers as c on u.id = c.user_id
+    JOIN orders as o on u.id = o.user_id;
+
```

Comment:
I dislike these blank lines in middle of tests. Reads like a separate test to me.

```suggestion
```